servers/rest-services/src/main/java/org/projectnessie/services/rest/ConstraintViolationExceptionMapper.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.services.rest;
+
+import javax.inject.Inject;
+import javax.validation.ConstraintViolation;
+import javax.validation.ConstraintViolationException;
+import javax.validation.ElementKind;
+import javax.validation.Path;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.Provider;
+import org.projectnessie.error.ErrorCode;
+import org.projectnessie.services.config.ServerConfig;
+
+/**
+ * "Special" implementation for exceptions that extend {@link ConstraintViolationException}, as
+ * those do not "go through" {@link NessieExceptionMapper}.
```

Comment:
should this mention that these exceptions should also not go through `ValidationExceptionMapper`  ?

Reply:
done